tests/is_balanced_or_bijected.py

```diff
@@ -0,0 +1,60 @@
+from supporting_components.graph_io import *
+from algorithms.color_refinement import color_refinement
+from algorithms.decide_gi import is_balanced_or_bijected
+
+
+# TODO: create a nice test that verifies the following:
```

Comment:
Is misschien wel handig om dit nu al te hebben? Dan kunnen we snel checken of de code ook het goede resultaat geeft?

Reply:
ik verwerk de test in een losse PR